articles/user-profile/user-impersonation.md

```diff
@@ -64,11 +65,11 @@ Let's assume that you have two apps, `app1` and `app2`, and you want to imperson
 
 ### Get a Token
 
-First, you have to generate a token that you can use to call the [Management APIv2](/api/management/v2), specifically the [Impersonation endpoint](/api/authentication/reference#impersonation). This token is called __Auth0 Management APIv2 Token__.
+First, you have to generate a token that you can use to call the [Management APIv2](/api/management/v2), specifically the [Impersonation endpoint](/api/authentication/reference#impersonation). This token is called the Auth0 Management APIv2 Token.
```

Comment:
The Impersonation endpoint is part of the Authentication API, not Management APIv2